MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ListBox.xaml

```diff
@@ -192,6 +193,10 @@
         </Setter>
     </Style>
 
+    <Style x:Key="MaterialDesignFlatToggleListBox" TargetType="{x:Type ListBox}" BasedOn="{StaticResource MaterialDesignToolToggleListBox}">
```

Comment:
I think we should call this ``` MaterialDesignToolToggleFlatListBox ``` to better indicate it as a variant of the underlying style.

Verbose, I know :)